tor/core/helpers.py

```diff
@@ -13,6 +13,7 @@
 import tor.core
 from tor.core import __version__
 from tor.core.config import config, Config
+from tor.core.config import SLACK_REMOVED_POST_CHANNEL_ID, SLACK_DEFAULT_CHANNEL_ID
```

Comment:
Do we need two import lines for this?

Reply:
Good catch, this happened during the refactoring